### DISCUSSION TITLE
path of downloaded pkgs via winget

### DISCUSSION THREAD

**Discussion by elswerky** [microsoft/winget-cli#2002] [General]
where can i find the files downloaded via winget 

whre is its path 

for example say i installed any app say google chrome via winget 

where can i find path of these downlaoded files 

for example in arch linux 
downlaoded pkgs are in /var/cache/pacman/archives 

here on windows ,, where is path of winget downl,oaded pkgs ??

**Comment by florelis** [comment]
Installers are downloaded to `%TEMP%\WinGet`, but they should be removed after the installation finishes. I believe in a previous version we used to use the temporary folder in the app data store.
